src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,100 @@ title = "Releases"
+The MetalLB version can be specified using `METALLB_VERSION=x.y.z`.
+* Support was added to force running `subctl verify` when testing end-to-end, ignoring any local tests.
+To use this feature, run `make e2e using=subctl-verify`.
+Verifications can be now specified using the `SUBCTL_VERIFICATIONS` flag, instead of relying on the default behavior.
+e.g.: `make e2e using=subctl-verify SUBCTL_VERIFICATIONS=connectivity,service-discovery`.
+* Support was added in Shipyard to easily create test clusters using OCP on top of AWS.
+This allows developers and users to easily stand up a test/PoC environment with Submariner using AWS+OCP.
+The deployed environment will use the latest published Submariner images, it's currently not possible to use locally built images.
+The following commands are supported:
+  * `make clusters using=aws-ocp`
+  * `make deploy using=aws-ocp`
+  * `make e2e using=aws-ocp`
+  * `make cleanup using=aws-ocp`
+* kubeconfig handling has been revamped to be consistent across all
+  `subctl` commands and to match `kubectl`’s behaviour.
+  * The single-context commands, `cloud-prepare`, `deploy-broker`, `export`,
+  `join`, `unexport` and `uninstall`, now all support a `--context` argument
+  to specify the kubeconfig context to use. kubeconfig files can be
+  specified using either the `KUBECONFIG` environment variable or the
+  `--kubeconfig` argument; `kubectl` defaults will be applied if
+  configured. If no context is specified, the kubeconfig default context
+  will be used.
+  * Multiple-context commands which operate on all contexts by default,
+  `show` and `gather`, support a `--contexts` argument which can be used
+  to select one or more contexts; they also support the `--context` argument
+  to select a single context.
+  * Multiple-context commands which operate on specific contexts,
+  `benchmark` and `verify`, support a `--context` argument to specify the
+  originating context, and a `--tocontext` argument to specify the target
+  context.
+  * `diagnose` operates on all accessible contexts by default, except
+  `diagnose firewall inter-cluster` and `diagnose firewall nat-traversal`
+  which rely on an originating context specified by `--context` and a
+  remote context specified by `--remotecontext`.
+  * Namespace-based commands such as `export` will use the namespace given
+  using `--namespace` (`-n`), if any, or the current namespace in the
+  selected context, if there is one, rather than the `default`
+  namespace.
+  * These commands also support all connection options supported by
+  `kubectl`, so connections can be configured using command arguments
+  instead of kubeconfigs.
+  * Existing options (`--kubecontext` etc.) are preserved for backwards
+  compatibility, but are deprecated and will be removed in the next
+  release.
+
+### Other changes
+
+* The flannel CNI is now properly identified during join.
```

Comment:
```suggestion
* The Flannel CNI is now properly identified during join.
```

The usages I see are very inconsistent, I see mixed cap and non-cap in every document I checked. If the project isn't consistent, I suggest we follow normal grammar rules and cap it as a proper noun. I'll propose adding a rule to our MD lint config to standardize.